<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="INBNRMARITA" failures="1" tests="2" name="Admin- Close SO" time="45.783" errors="0" timestamp="2022-07-25T16:20:43 IST">
  <testcase classname="script.NewClosePoConsumer" name="poClose" time="8.332"/>
  <testcase classname="script.NewClosePoConsumer" name="closePO" time="31.383">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at script.NewClosePoConsumer.closePO(NewClosePoConsumer.java:93)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- closePO -->
  <testcase classname="script.NewClosePoConsumer" name="companyName" time="0.0">
    <ignored/>
  </testcase> <!-- companyName -->
  <testcase classname="script.NewClosePoConsumer" name="validateCropList" time="0.0">
    <ignored/>
  </testcase> <!-- validateCropList -->
  <testcase classname="script.NewClosePoConsumer" name="resetButton" time="0.0">
    <ignored/>
  </testcase> <!-- resetButton -->
</testsuite> <!-- Admin- Close SO -->
